src/views/Home/components/LotteryBanner.tsx

```diff
@@ -38,7 +38,9 @@ const LeftWrapper = styled(Flex)`
   flex-direction: column;
   flex: 1;
   padding-bottom: 40px;
-  padding-top: 40px;
```

Comment:
Reinstate this please memoyil - it can be changed to 24px though

Not related to your change - but could you give `RightWrapper` a margin top of 24px only below `theme.mediaQueries.sm` too please?

Reply:
Done